Site1.Master.cs

```diff
@@ -11,7 +11,45 @@ public partial class Site1 : System.Web.UI.MasterPage
     {
         protected void Page_Load(object sender, EventArgs e)
         {
+            try
+            {
+                if (Session["role"].Equals(""))
+                {
+                    LinkButton1.Visible = true; //user login link button
+                    LinkButton2.Visible = true; //sing up link button
 
+                    LinkButton3.Visible = false; //logout link button
+                    LinkButton7.Visible = false; //hello user login link button
+
+                    LinkButton6.Visible = true; //admin login link button
+                }
+                else if (Session["role"].Equals("user"))
+                {
+                    LinkButton1.Visible = false; //user login link button
```

Comment:
Again, please name your input elements with proper and readable names. It's easier for you to go over your code and edit rather than having to refer back and forth all the time. 

```suggestion
                    **lbUserLogin**.Visible = false;
```